docs/day-2-operations/Backups.adoc

```diff
@@ -144,8 +144,105 @@ Backup the dump.rb file from redis
 oc cp $(oc get pods -l 'deploymentConfig=system-redis' -o json | jq '.items[0].metadata.name' -r):/var/lib/redis/data/dump.rdb ./system-redis-dump.rdb
 ----
 
+=== `Secrets`
+[source,bash]
+----
+oc get secrets system-smtp -o json --export > system-smtp.json
```

Comment:
For both secrets and configmaps it seems that the --export flag does not remove possible ownerReference elements. Can we modify the commands to remove them?
For example with:
```
 oc get secrets system-smtp -o json --export > system-smtp.json | jq -r 'del(.metadata.OwnerReferences)'
```
It should work even if ownerReferences do not exist. This adds 'jq' as a prerequisite tool

Reply:
I've also observed that the .metdata.selfLink field also keeps being set:
```
msoriano@localhost:~/go/src/github.com/3scale/3scale-operator/mybackup/secrets (3scale-2.8-stable-prod)$ cat system-database.json 
{
   ...
  "metadata": {
    ...
    "selfLink": "/api/v1/namespaces/msoriano-backuprestore/secrets/system-database"
  },
  "type": "Opaque"
}
```

Which seems to contain some kind of reference to a namespace. Can we remove it just in case too? If we build it from my previous comment we could delete both ownerReferences and selfLink with:
```
oc get secrets system-smtp -o json --export | jq -r 'del(.metadata.selfLink)' > system-smtp.json
```